pkg/transport/message.go

```diff
@@ -54,7 +54,7 @@ type transporterImpl struct {
 
 func (impl *transporterImpl) Send(m msg.Message) error {
```

Comment:
We intend to add a context parameter similar to the Do function to allow cancellation of sending.

One case is when the user exits manually, which can be canceled through this mechanism, and another case is for potential scenarios where a timeout might be set later.

Reply:
Adding context parameters would be a better solution.
In other cases, adding a timeout is indeed a viable solution, and I have considered using this pattern before